tds-dll-schemas/src/main/resources/import/genericsbacconfig/db/migration/V4__spanish_message_translations.sql

```diff
@@ -0,0 +1,135 @@
+/***********************************************************************************************************************
```

Comment:
These are picked up as part of the DB script?

Reply:
Yep, the `db-schema-setup.sh` script runs through this entire directory in order: https://github.com/SmarterApp/TDS_Build/blob/develop/database/tds/db-schema-setup.sh#L236